### DISCUSSION TITLE
How to place sign with text?

### DISCUSSION THREAD

**Discussion by MrlnHi** [Minestom/Minestom#664] [Help] [answered]
How do I place a sign with text in an instance in code?

**Comment by emortaldev** [comment]
I assume something with Block.SIGN.withNBT or with properties, not really sure I might look into it later

**Reply by MrlnHi** [reply]
it's fine already, as stated in #665

**Comment by TheMode** [comment, marked as answer]
Fixed by 502db8ec5ceb66c074247966cfc6597c1647aac9
